### ISSUE TITLE
Out-of-date vendor dependencies are causing downstream builds to fail

### ISSUE THREAD

**Issue by jimmyocto** [cortesi/devd#94]
TL;DR - please update your vendor dependencies for `devd` and `modd`! Out of date dependencies are causing downstream project builds to fail! (potentially only for OSX users)

Hi @cortesi , I'd like to bring to your attention that one of your dependencies, https://github.com/rjeczalik/notify has updated some `cgo` constructs in their repository https://github.com/rjeczalik/notify/pull/162. The `vendor` folder in this repo is using an out-of-date version of this dependency. 

For some reason this is causing all attempts to download and build projects using `devd` and `modd` to fail.

```
$ go get github.com/cortesi/devd/cmd/devd
# github.com/cortesi/devd/vendor/github.com/rjeczalik/notify
go/src/github.com/cortesi/devd/vendor/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:51: cannot use nil as type _Ctype_CFAllocatorRef in argument to func literal
go/src/github.com/cortesi/devd/vendor/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:165: cannot use nil as type _Ctype_CFAllocatorRef in argument to _Cfunc_CFStringCreateWithCStringNoCopy
go/src/github.com/cortesi/devd/vendor/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:166: cannot use nil as type _Ctype_CFAllocatorRef in argument to func literal
```

I was able to fix the issue by forking `modd` and `devd` and updating dependencies with

```
dep ensure -update
```

and you can see the results of my changes here:
https://github.com/jimmyocto/devd
https://github.com/jimmyocto/modd

However, I recommend that you update the dependencies yourself as I ran into some additional import issues working out of a forked repo (my `devd` was still referencing your `devd` so I had to fix that here https://github.com/jimmyocto/devd/commit/74ee140080eb683367a17a58e6d056bf5efcf2e6 )

You can see that this issue is impacting other users here: https://github.com/cortesi/modd/issues/57
I can confirm this `cgo` issue is happening for OSX users but I do not know if this issue can be easily replicated on Linux or Windows.

**Comment by cortesi** [comment]
Thanks! This should now be fixed.
